### PR TITLE
Resource overview section added

### PR DESCRIPTION
**Pull request by Atiksh04**
I've added changes for the resource overview section with the following sub-sections:
- Alternatives
- Comparisons
- Analytics Review
- Analytics Pricing
- Changes for Translation (locale files and route changes in nuxt config)
Have added subpages to these sections as per the requirement.

### CODE REVIEW

File: pages/resources/alternatives/index.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  }
+  return {};
+});
+
+const { articles, pending, error } = await useArticle({
+  routeName: "resources-alternatives",
+  articleType: "resources-alternatives",
+});
+
+const getFlagUrl = (locale) => {
```
Comment (adriaanvanrossum) [inline pages/resources/alternatives/index.vue:78]: We could move this function to `utils/miscellaneous.js`, after that, you can use it anywhere without importing. I already have it copied, but 3 times is a bit too much.

File: pages/resources/analytics-pricing/index.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  }
+  return {};
+});
+
+const { articles, pending, error } = await useArticle({
+  routeName: "resources-analytics-pricing",
+  articleType: "resources-analytics-pricing",
+});
+
+const getFlagUrl = (locale) => {
```
Comment (adriaanvanrossum) [inline pages/resources/analytics-pricing/index.vue:77]: Same, use `utils/miscellaneous.js`

File: pages/resources/analytics-review/index.vue [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  }
+  return {};
+});
+
+const { articles, pending, error } = await useArticle({
+  routeName: "resources-analytics-review",
+  articleType: "resources-analytics-review",
+});
+
+const getFlagUrl = (locale) => {
```
Comment (adriaanvanrossum) [inline pages/resources/analytics-review/index.vue:77]: Same, use `utils/miscellaneous.js`

**Review by adriaanvanrossum — CHANGES_REQUESTED** [review]
It looks already excellent, I just have a few nitpick things. Some code is double (and I know, I did have that as well), but if we work with the two of us in the same code base, it's nicer to not have (too much) duplicate logic. Using prettier will fix some code styling "issues" as well, so we have a solution for that as well.

Don't forget to create a PR for Strapi to add the hard-coded article types.

File: pages/resources/alternatives/index.vue (hunk 1 above)
Comment (Atiksh04) [reply pages/resources/alternatives/index.vue]: I have updated this, please have a look

File: pages/resources/analytics-pricing/[slug].vue [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<template>
+<Article
```
[the remark this answers is not in the document]
Comment (Atiksh04) [reply pages/resources/analytics-pricing/[slug].vue]: I have updated this, please have a look

File: pages/resources/analytics-pricing/index.vue (hunk 2 above)
Comment (Atiksh04) [reply pages/resources/analytics-pricing/index.vue]: Updated this, please have a look

File: pages/resources/analytics-review/index.vue (hunk 3 above)
Comment (Atiksh04) [reply pages/resources/analytics-review/index.vue]: This is also updated

File: pages/resources/alternatives/index.vue (hunk 1 above)
Comment (Atiksh04) [reply pages/resources/alternatives/index.vue]: have resolved this

**Pull request merged**